### PR TITLE
docs: add description about sort-by method

### PR DESCRIPTION
**Pull request by NozomuIkuta**
## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

None of above: update on docs

## Description

This PR is based on #695 and is to add description about behavior of `sortBy` method, that is, the sorting is case sensitive.
I also added a snippet by which to work around the behavior.

For more detail, please refer to #695.

## Checklist:

- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes (if not applicable, please state why)

In my local, I ran `yarn run dev` in `docs` directory and checked UI.

### CODE REVIEW

File: docs/content/en/snippets.md [hunk 1]
```diff
@@ -148,6 +148,48 @@ even when the current page is showing higer-position document.
...
+
+It is needed to work around Nuxt Content's case-sensitive sorting, to add extra properties to documents, whose value is the lower-cased.
+
+```javascript[nuxt.config.js]
+export default {
+  hooks: {
+    'content:file:beforeInsert': (document) => {
+      if (document.extension === '.md') {
+        Object.entries(document).forEach(([key, value]) => {
+          const _key = `case_insensitve__${key}`; // prefix is arbitrary
```
Comment (Atinux) [inline docs/content/en/snippets.md:161]:
```suggestion
          const _key = `case_insensitive__${key}`; // prefix is arbitrary
```

File: docs/content/en/snippets.md [hunk 2]
```diff
@@ -148,6 +148,48 @@ even when the current page is showing higer-position document.
...
+      .sortBy('case_insensitive__title', 'asc') // Set prefixed prop
+      .fetch()
+
+    return {
+      articles
+    }
+  }
+}
+</script>
+```
```
Comment (Atinux) [inline docs/content/en/snippets.md:189]:
```suggestion
```ts
export default {
  async asyncData({ $content, params }) {
    const articles = await $content('articles', params.slug)
      .sortBy('case_insensitive__title', 'asc') // Set prefixed prop
      .fetch()

    return {
      articles
    }
  }
}
```
```

File: docs/content/en/snippets.md [hunk 3]
```diff
@@ -148,6 +148,48 @@ even when the current page is showing higer-position document.
 
 > Check out the [surround documentation](/fetching#surroundslug-options)
 
+### Case-Insensitive Sorting
+
+It is needed to work around Nuxt Content's case-sensitive sorting, to add extra properties to documents, whose value is the lower-cased.
+
+```javascript[nuxt.config.js]
```
Comment (Atinux) [inline docs/content/en/snippets.md:155]:
```suggestion
```js [nuxt.config.js]
```

**Comment by Atinux** [comment]
Looking good, I added a somme code suggestions 😊

**Comment by NozomuIkuta** [comment]
Thank you for suggestions and fix a typo!
I accepted all of them. 👍

**Review by Atinux — APPROVED** [review]

**Pull request merged**